localisation/00_wc_artifact_localisation.csv

```diff
@@ -72,6 +72,8 @@ gorehowl_desc;A massive axe wielded by Grommash Hellscream of the Warsong Clan.
 warblade_of_archimonde;Warblade of Archimonde;;Kriegsklinge von Archimonde;;;;;;;;;;;x
 warblade_of_archimonde_desc;This humongous warblade belongs to Archimonde the Defiler, Lord of the Burning Legion. Countless worlds were conquered by the might of this blade and its owner.;;Diese gigantische Kriegsklinge geh�rt Archimonde dem Entweiher, dem Herrn der brennenden Legion. Unz�hlige Welten wurden durch die Macht dieser Klinge und ihres Besitzers erobert.;;;;;;;;;;;x
 # Epic #;;;;;;;;;;;;;;
+blightreaper;Blightreaper;;Seuchenschnitter;;;;;;;;;;;x
+blightreaper_desc;This scythe, infused with magic, causes death and destruction to befall the enemy. That the kingdom may be rid of all invaders.;;Diese mit Magie durchwirkte Sense l�sst Tod und Verderben �ber die Feinde hereinbrechen. Auf dass das K�nigreich von s�mtlichen Eindringlingen befreit werde.;;;;;;;;;;;x
```

Comment:
Joined the two sentences together, as the second sentence was a fragment.
```suggestion
blightreaper_desc;This scythe, infused with magic, causes death and destruction to befall the enemy, so that the kingdom may be rid of all invaders.;;Diese mit Magie durchwirkte Sense l�sst Tod und Verderben �ber die Feinde hereinbrechen. Auf dass das K�nigreich von s�mtlichen Eindringlingen befreit werde.;;;;;;;;;;;x
```

Reply:
done